module/platform/Webots/data/config/WebotsCameras/left_camera.yaml

```diff
@@ -23,9 +23,9 @@ lens:
   fov: 2.0944
   # The homogenous transform from the rigid platform this camera is attached to to its optical location
   Hpc:
-    - [1.0, 0.0, 0.0, 0.0889361]
-    - [0.0, 1.0, 0.0, 0.0352991]
-    - [0.0, 0.0, 1.0, 0.0713675]
+    - [0.0, 0.0, 1.0, 0.0889361]
+    - [1.0, 0.0, 0.0, 0.0352991]
+    - [0.0, 1.0, 0.0, 0.0713675]
```

Comment:
wait I just realised why you changed all the cameras. can't we just rotate in code somewhere so we don't do this? Our calibration tool/s won't work anymore if we have it like this. and if we can't calibrate the cameras (extrinsics) then localisation will be off.

Reply:
How is `Hpc` calculated?

Reply:
You'd have to check the code for the calibration tool.

Reply:
I think it is given quantity to calibration so we either calibrate all the cameras again with an updated Hpc, or we apply this rotation offset to the old Hpc.